src/plugins/intel_gpu/src/runtime/memory_pool.cpp

```diff
@@ -168,21 +220,30 @@ memory::ptr memory_pool::get_from_padded_pool(const layout& layout,
                 rec_list._memory->get_layout().format != format::fs_b_yx_fsv32 &&
                 layout.format != format::fs_b_yx_fsv32 &&
                 !has_conflict(rec_list._users, restrictions, network_id)) {
-                rec_list._users.insert({unique_id, network_id, prim_id});
+                rec_list._users.insert({unique_id, network_id, prim_id, layout.bytes_count()});
```

Comment:
Could we do this only when DumpMemoryPool is set? 
Beause calculating bytes_count can be another bottleneck too.

Reply:
Done.